Notation.Plugin.AzureKeyVault.Tests/KeyVault/KeyVaultClientTests.cs

```diff
@@ -0,0 +1,253 @@
+using System;
+using System.IO;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Security.KeyVault.Certificates;
+using Azure.Security.KeyVault.Keys;
+using Azure.Security.KeyVault.Keys.Cryptography;
+using Azure.Security.KeyVault.Secrets;
+using Moq;
+using Xunit;
+using Notation.Plugin.Protocol;
+using System.Text;
```

Comment:
nit: it would be better to sort those imports.

Reply:
Updated.